selectors/i18n.js

```diff
@@ -3,12 +3,17 @@
 
 import {getConfig} from 'mattermost-redux/selectors/entities/general';
 import {getCurrentUserLocale} from 'mattermost-redux/selectors/entities/i18n';
+import {General} from 'mattermost-redux/constants';
 
 import * as I18n from 'i18n/i18n';
 
 // This is a placeholder for if we ever implement browser-locale detection
 export function getCurrentLocale(state) {
-    return getCurrentUserLocale(state, getConfig(state).DefaultClientLocale);
+    const currentLocale = getCurrentUserLocale(state, getConfig(state).DefaultClientLocale);
+    if (I18n.isLanguageAvailable(currentLocale)) {
```

Comment:
shouldn't we check that there is a value in currentLocale?
```suggestion
if (currentLocale && I18n.isLanguageAvailable(currentLocale)) {
```

Reply:
`getCurrentUserLocale` already does a null check, it will return `DEFAULT_LOCALE` if the user's locale is null or empty.